synapse_domain_rule_checker/__init__.py

```diff
@@ -143,7 +143,14 @@ async def _user_may_invite(
             Whether the invite can be allowed to go through.
         """
 
-        if self._config.can_only_invite_during_room_creation:
+        inviter_domain = self._get_domain_from_id(inviter_userid)
+
+        if (
+            self._config.can_only_invite_during_room_creation
+            # only apply to room created locally,
+            # invites coming through fed should still go further
```

Comment:
```suggestion
            # Only check if the room is new for rooms created locally, because we
            # can't reliably figure out whether a remote invite is for a new room.
```